### ISSUE TITLE
ERROR:  couldn't find TuplestoreScan node in combiner's plan

### ISSUE THREAD

**Issue by dizer** [pipelinedb/pipelinedb#1798]
Ubuntu 16.04, Pipelinedb 0.9.7

After upgrading from 0.9.6 (pipelinedb_dump -F c => pg_restore) getting this on insert in stream:

```
LOG:  starting pipelinedb processes for database: "dbname"
LOG:  pipelinedb process "queue0 [dbname]" running with pid 32339
LOG:  pipelinedb process "reaper0 [dbname]" running with pid 32338
LOG:  pipelinedb process "worker0 [dbname]" running with pid 32341
LOG:  pipelinedb process "combiner0 [dbname]" running with pid 32340
ERROR:  couldn't find TuplestoreScan node in combiner's plan
LOG:  worker process: combiner0 [dbname] (PID 32340) exited with exit code 1
LOG:  pipelinedb process "combiner0 [dbname]" running with pid 32431
ERROR:  couldn't find TuplestoreScan node in combiner's plan
LOG:  worker process: combiner0 [dbname] (PID 32431) exited with exit code 1
LOG:  pipelinedb process "combiner0 [dbname]" running with pid 32434
```

Any idea what to do?

**Comment by derekjn** [comment]
@dizer can you show us the continuous view definition that is causing this error? If you don't know which one, can you share all of them?

**Comment by dizer** [comment]
Even simple view like ```select sum(x) from stream``` caused this problem. So I've dropped db and import raw data.

**Comment by dizer** [comment]
Looks like problem related to CONTINUOUS TRANSFORM count. Background worker crashed if stream has ~20 (in my case) transforms on it. 

Transforms like:

```sql
CREATE CONTINUOUS TRANSFORM sample_stream_transform AS SELECT name FROM ONLY events_stream
THEN EXECUTE PROCEDURE pipeline_stream_insert(sample_stream);
```

**Comment by derekjn** [comment]
@dizer We haven't been able to reproduce this with many more than 20 transforms, can you share some SQL to reproduce the issue? Also, what is the output of `SELECT pipeline_version();`?

**Comment by dizer** [comment]
```
PipelineDB 0.9.7 at revision 921d7185ff14a8e44ccc8934d517685549e5567a on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 5.4.0-6ubuntu1~16.04.4) 5.4.0 20160609, 64-bit
```

Problem appeared after migration to 0.9.7 and delta streams.

**Comment by derekjn** [comment]
Thanks @dizer. Any SQL you could share would be helpful.

**Comment by derekjn** [comment]
@dizer this issue should be fixed now, please see #1811.
